common/go-utils/src/inv.rs

```diff
@@ -237,4 +256,41 @@ mod tests {
             UnsupportedOsError(..)
         ));
     }
+
+    impl Default for Artifact {
+        fn default() -> Self {
+            Self {
+                go_version: String::from("go1.7.2"),
+                semantic_version: Version::parse("1.7.2").unwrap(),
+                os: Os::Linux,
+                arch: Arch::X86_64,
+                url: String::from("foo"),
+                sha_checksum: String::from("bar"),
+            }
+        }
+    }
```

Comment:
interesting, it never occurred to me to add an implementation in a `test` configuration.  this would cause some unexpected conflict though if, at some later point, `Default` was implemented on `Artifact` in the release code.  should you perhaps do this instead?

```suggestion
    fn create_artifact() -> Artifact {
        Artifact {
            go_version: String::from("go1.7.2"),
            semantic_version: Version::parse("1.7.2").unwrap(),
            os: Os::Linux,
            arch: Arch::X86_64,
            url: String::from("foo"),
            sha_checksum: String::from("bar"),
        }
    }
```

Reply:
I haven't seen that approach used elsewhere either tbh :) At any rate, I've changed the default Artifact impl to `create_artifact` as you suggested in https://github.com/heroku/buildpacks-go/pull/230/commits/9acf2d6f52dacfed041f3b0be2b5ace82bb2b354